### ISSUE TITLE
Use localised number formats for transaction details

### ISSUE THREAD

**Issue by yoshimo** [bisq-network/bisq#3145]
When an offer is confirmed by both sides and you get the bank account , identifier for the transaction and the rest of the necessary payment details, the amount of money to pay is written with a "."  as separator.
In some parts of the world we dont use a dot but a "," to tell dollars and cents, euro and cents etc apart. If you copy & paste one version into your online banking website you either get errors or you might paste a bit too much because of 1.000 could also mean one thousand and not just plain simply exactly one .

Would be nice if that part of bisq would be locale dependent too.

**Comment by ghost** [comment]
Can you please provide some example in which currency I can achieve this situation. 

My experience is, that Bisq *always* (AFAIK) rounds payment to the whole number => no cents.

Do I miss something?

Thank you.

**Comment by yoshimo** [comment]
Yes the final sum is rounded for privacy reasons, still the exchange rate and the other fields , be it btc, the exchange rate or the range between the lowest amount and the highest you can buy do show fractions. 
So can the fee estimations " 0.00023423 BTC (~13.06 EUR)
"8.50 bsq (~5.12 EUR)
Depending on the btc-fiat exchange rate 8785.0000  could also be interpreted in the wrong way depending on your local habits.

**Comment by yoshimo** [comment]
![grafik](https://user-images.githubusercontent.com/17909/82139847-3b0dd780-982b-11ea-9b08-6430d98fdec9.png)

**Comment by cd2357** [comment]
This could be achieved by the user choosing a specific locale.

Example from https://www.baeldung.com/java-decimalformat :

```
assertThat(new DecimalFormat("#,###.##", 
	  new DecimalFormatSymbols(Locale.ENGLISH)).format(d))
	  .isEqualTo("1,234,567.89");

assertThat(new DecimalFormat("#,###.##", 
	  new DecimalFormatSymbols(Locale.ITALIAN)).format(d))
	  .isEqualTo("1.234.567,89");
```

The locale could then affect the rendering of all numbers, by choosing the correct decimal separator when rendering.

**Comment by premek** [comment]
Note: also see https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#getDefault--

**Comment by alvasw** [comment]
Hey everyone!
I'm working on this now. If anyone else claimed this already just let me know. Thanks!

**Comment by alvasw** [comment]
I'm busy working on Bisq 2. Feel free to base your solution on my pull request if it's helpful.
